api/base.proto

```diff
@@ -137,6 +137,14 @@ message ApiConfigSource {
 message AggregatedConfigSource {
 }
 
+message KdsSecretConfig {
```

Comment:
Note: this landed in `base.proto` and not in `kds.proto`, because there was a cycle between `tls_context.proto` and `kds.proto`.